components/camel-sql/src/main/java/org/apache/camel/processor/aggregate/jdbc/JdbcCamelCodec.java

```diff
@@ -35,6 +37,12 @@
 public class JdbcCamelCodec {
 
     public byte[] marshallExchange(CamelContext camelContext, Exchange exchange, boolean allowSerializedHeaders) throws IOException {
```

Comment:
With these original methods (called solely by another class in package) I as tempted to ...
1. `@Deprecate` 
2. reduce to package access
3. or delete and inline into single known caller
... but didn't do this as their reusable existence is what lead me here in the first place!